ironic/drivers/modules/drac/raid.py

```diff
@@ -996,6 +1064,37 @@ def _create_virtual_disks(task, node):
     return _commit_to_controllers(node, controllers)
 
 
+def _config_raid_settings(node):
+    raid_settings = list_raid_settings(node)
+    controllers = list_raid_controllers(node)
+    raid_controller = [cntrl for cntrl in controllers
+                       if cntrl.model.startswith('PERC H740P')][0]
```

Comment:
I know it's not strictly necessary, but I think it would be nice to:
if client.is_raid_controller(cntrl.id, controllers) and cntrl.model.startswith('PERC H740P')

Reply:
It would be nicer to check to see if the controller supports EHBA mode instead of relying on comparing the model number to a hard coded string.  This way, we won't have to update this code when a new RAID controller comes out that supports EHBA mode.

Reply:
so the above would look something like:
if client.is_raid_controller(cntrl.id, controllers) and client.controller_supports_ehba_mode(cntrl.id)

Reply:
I don't know if it's possible to have multiple H740Ps in a server, but we should probably handle it just in case.
So above, we would build a list of all H740P controllers, not just the first, and below, we would loop through the list.

Reply:
Yes you have a valid point Chris, but above in second comment you have mention another condition to validate controller having ehba mode, that is `client.controller_supports_ehba_mode(cntrl.id)` which is not implemented in python-dracclient.
Do you want me to add this?

Reply:
Hmmmm...  Thinking about it some more, I think we should add that method on the ironic side of things in raid.py.

Reply:
Done